### ISSUE TITLE
jQuery 1.5.2 and Google CDN

### ISSUE THREAD

**Issue by peterbe** [mozilla/playdoh#24]
The "recommended way" to serve jQuery as per the [HTML5 Boilerplate by Paul Irish](http://html5boilerplate.com/) is:

```
<script src="//ajax.googleapis.com/ajax/libs/jquery/1.5.2/jquery.min.js"></script>
<script>window.jQuery||document.write('<script src="js/libs/jquery-1.5.2.min.js">\x3C/script>');</script>
```

Having said that, it's preferred to use a local copy when doing development because web devs reload a lot. Not sure how to solve this seamlessly. 

The Google CDN has the advantage that a lot people have it cached already and the simple fact that it's a proper CDN.
